Makefile

```diff
@@ -77,7 +77,8 @@ METADATA_VAR += CommitSHA=$(EXTRA_VERSION)
 METADATA_VAR += BaseDockerLabel=$(BASE_DOCKER_LABEL)
 METADATA_VAR += DockerNamespace=$(DOCKER_NS)
 
-GO_VER = 1.24.0
+GO_VER := $(shell grep '^go[ \t]' < go.mod)
```

Comment:
Note that this cannot use `go list -m` to obtain the version from the go directive since it will fail to parse the module file if the local Go version is behind the module's required Go version. This results in a less helpful error message from `check_go_version.sh`:

```text
ERROR:  is required to build Fabric and you are using 1.23.5. Please update go.
```

Using the approach implemented here, the error message is more helpful:

```text
ERROR: 1.24.0 is required to build Fabric and you are using 1.23.5. Please update go.
```